56.md

```diff
@@ -2,35 +2,37 @@
 NIP-56
 ======
 
-Reporting
+Labeling
 ---------
 
-`draft` `optional` `author:jb55`
+`draft` `optional` `author:jb55` `author:michaelhall923` `author:gruruya`
 
-A report is a `kind 1984` note that is used to report other notes for spam,
-illegal and explicit content.
+A label is a `kind 1984` note that describes other notes or pubkeys.  
 
 The content MAY contain additional information submitted by the entity
-reporting the content.
+labeling the content.
 
 Tags
 ----
 
-The report event MUST include a `p` tag referencing the pubkey of the user you
-are reporting.
+The label event contains one or more tags:
+- `[e, <event-id>, <label>]` - Label event
+- `[p, <pubkey>]` - Use pubkey to find recommended relays
+- `[p, <pubkey>, <label>]` - Same as above, but also label pubkey
 
-If reporting a note, an `e` tag MUST also be included referencing the note id.
+The `<label>` may optionally be followed by a `<classification>` such as:
+- `"content"` - The content is `label`
+- `"image"` - The image depicts `label`
+- `"topic"` - The event/profile is about `label`  
 
-A `report type` string MUST be included as the 3rd entry to the `e` or `p` tag
-being reported, which consists of the following report types:
+Following the classification, there may be an `<affirmative marker>` which must be one of the following:
```

Comment:
I don't understand what this is. Does this mean "The image is not a picture of Belgium"?
`[ "e", <event id 1>, "belgium", "image", "-"]`

Reply:
Yes. It's a way for users to vote against a label. You're the second person asking so I need to clarify the wording more.

Reply:
I'm changing the name, `affirmative marker` is too unknown. Which of these seems clearest to you:
- stance
- alignment
- polarity

Reply:
To be honest, I think `affirmative marker` is actually the best choice. It's the closest to the noun `confirmation`. Or use `confirmation marker` (?)

Maybe it would suffice to describe the examples in a few sentences.

For example:

```json
{
  "kind": 1984,
  "tags": [
    [ "p", <pubkey>, "nudity"] // pubkey to find recommended relays. relays contain events about nudity
    [ "e", <event id 1>, "belgium", "image", "-"], // event is labeled as not containing an image of belgium
    [ "e", <event id 1>, "france", "image"], // event is labeled as containing an image of france
    [ "e", <event id 2>, "my favorite"], // event is labeled as "my favorite"
    [ "p", <pubkey>] // pubkey to find (other) recommended relays
  ],
  "content": "",
  ...
}
```

I don't like how the JSON syntax is malformed though. The red highlighting is hard to look at.

Reply:
I'll split that one into a block followed by an explanation.

Reply:
I prefer vote but am kinda indifferent. I guess I just struggle to see the use case for this tbh. Are we expecting that clients will want to implement an "unreport" feature or something?

Reply:
Yeah, an "unlabel," I don't see it being used for reports too much as well. It's can be used for other things like if an account is a celebrity or an image is of belgium vs france

Reply:
Ok. I personally would probably never use it but I guess it doesn't hurt. It's good that you put the default state of "content" and "+" because that's how most people will probably use this.

Reply:
Yeah and for backwards compatability with reports